pkg/cover/cover.go

```diff
@@ -126,7 +126,9 @@ func Execute(args, newGopath, target, mode, agentPort, center string) error {
 		log.Errorf("Target directory %s not exist", target)
 		return ErrCoverPkgFailed
 	}
-
+	if mode == "" {
```

Comment:
This is a common lib, so it is not friendly to set a default value by itself. suggest to return err if mode equals empty.

Reply:
if mode flag is not setting, func can't get default cover mode value. So need to set a default value in the func